build/reqpacks/system-setup.py

```diff
@@ -29,13 +29,17 @@ def debian_compat(self):
         self.install("build-essential")
         self.install("python3-psutil")
         self.install("libsqlite3-dev")
+        self.install("unixodbc-dev")
 
     def redhat_compat(self):
         self.run("%s/bin/enable-utf8" % READIES)
 
         self.group_install("'Development Tools'")
         self.install("redhat-lsb-core")
         self.install("libsqlite3x-devel")
+        self.install("gcc")
+        self.install("gcc-c++")
```

Comment:
It's better to do the following here: `self.run("%s/bin/getgcc --modern" % READIES)`

Reply:
But RedisGears do pip install which does the cpp.

Reply:
This is CentOS, and the default gcc is old. `getgcc --modern` gets a newer gcc.